src/DotLiquid.Tests/TemplateTests.cs

```diff
@@ -8,47 +8,56 @@ namespace DotLiquid.Tests
     [TestFixture]
     public class TemplateTests
     {
+
+        private System.Collections.Generic.List<string> Tokenize(string input)
+        {
+            var v20 = Tokenizer.Tokenize(input, SyntaxCompatibility.DotLiquid20);
+            var v22 = Tokenizer.Tokenize(input, SyntaxCompatibility.DotLiquid22);
+            Assert.AreEqual(v20, v22);
+            return v22;
+        }
+
         [Test]
         public void TestTokenizeStrings()
         {
-            CollectionAssert.AreEqual(new[] { " " }, Template.Tokenize(" "));

```

Comment:
Keep the class prefix for clarity, especially now that there are multiple Tokenize methods in this context.

Reply:
Template.Tokenize doesn't exist any more. It is a separate class Tokenizer.Tokenize now - and that expects a specific syntax as an argument. Hence the Tokenize above that actually tests multiple versions against each other.

Reply:
So here `TemplateTests.Tokenize(` would make it clear which Tokenize is at play.

Reply:
I'll rename it to TokenizeMultiVersions? Will that be clearer what it is doing and also deal with your issue?

Reply:
Sure, that will remove ambiguity. Personally I always put prefixes for clarity, but that's fine when at least the method name doesn't overlap.